c++/src/capnp/rpc-twoparty.c++

```diff
@@ -150,8 +157,12 @@ public:
 
     network.previousWrite = KJ_ASSERT_NONNULL(network.previousWrite, "already shut down")
         .then([this]() {
-      return kj::evalNow([&]() { return network.getStream().writeMessage(fds, message); })
-          .catch_([this](kj::Exception&& e) {
+      return kj::evalNow([&]() {
+        created = network.clock.now();
```

Comment:
Hm? Unless I'm misreading, isn't this too late to be setting `created`? We may have had to wait quite some time on `previousWrite` to resolve. I'd expect this variable to be set when `send()` is called, not after `previousWrite` is resolved.

We also don't really need `created` to be a member variable anymore, since it can just be declared as a local in `send()` and captured by the lambda that sets `network.currentOutgoingMessageCreationTime`. Although that doesn't really matter.

Reply:
Yeah we need to be setting `created` at the beginning of the `send()` method. (Maybe it should be called `sendTime`...)